### ISSUE TITLE
[bug] text in nested tspans gets removed

### ISSUE THREAD

**Issue by tobwen** [benboba/svg-slim#27]
## summary
Text in nested tspans gets removed when using defaults. I'm using v1.5.3

## incoming SVG file (Inkscape v1.0.1 - plain-svg-output)
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" version="1.0" width="800" height="600" id="svg2020">
    <text xml:space="preserve" style="font-size:24.000000px;font-style:normal;font-variant:normal;font-weight:700;font-stretch:Normal;line-height:125.000000%;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;font-family:sans-serif;text-align:start;text-anchor:start;" transform="matrix(1.000000,-0.000000,0.000000,1.000000,200,200)" x="0.000000" y="0.000000" id="text2018">
        <tspan x="0.000000" y="0.000000" id="tspan2016">
            <tspan dx="0.000000" dy="0.000000" style="fill:#000000;font-size:24.000000px;font-style:normal;font-variant:normal;font-weight:700;font-stretch:Normal;font-family: sans-serif;" id="tspan2014">fixme</tspan>
        </tspan>
    </text>
</svg>
```

## what slimming looks like
```svg
<?xml version="1.0"?>
<svg xmlns="http://www.w3.org/2000/svg" width="8e2" height="6e2">
  <text style="font-size:24;font-weight:700;font-stretch:Normal;letter-spacing:0;word-spacing:0;font-family:sans-serif" dx="2e2" dy="2e2">
    <tspan x="0" y="0"> </tspan>
  </text>
</svg>
```

## what it should look like
```svg
<?xml version="1.0"?>
<svg xmlns="http://www.w3.org/2000/svg" width="8e2" height="6e2">
  <text style="font-size:24;font-weight:700;font-stretch:Normal;letter-spacing:0;word-spacing:0;font-family:sans-serif" dx="2e2" dy="2e2">fixme</text>
</svg>
```

**Comment by benboba** [comment]
According to the svg specification, tspan nodes should not be nested in tspan. This may be due to the browser's compatibility work, which caused unexpected removal. I will fix this problem in version 2.0.0

**Comment by benboba** [comment]
2.0.0 has been released. Starting from this version, the package name is changed to svg-slim, and the npm address is https://www.npmjs.com/package/svg-slim

**Comment by tobwen** [comment]
> According to the svg specification, tspan nodes should not be nested in tspan. This may be due to the browser's compatibility work, which caused unexpected removal. I will fix this problem in version 2.0.0

You're right. _Internet Explorer 11_ needed **one** of the TSPANs. I'm gonna write about it soon.
